src/shared/proto/BUILD

```diff
@@ -1,12 +1,15 @@
 load("@rules_proto//proto:defs.bzl", "proto_library")
 
+# TODO: more sane location for this file
```

Comment:
🚀  Can we get a more descriptive comment for this TODO haha

/Issue#

Reply:
Oh this is the sane location. Removed the comment